test/dynamo/test_unspec.py

```diff
@@ -341,6 +341,16 @@ def fn(inputs, dim):
         compl_fn = torch.compile(fn, dynamic=True, backend="eager", fullgraph=True)
         self.assertEqual(compl_fn(inputs, dim), fn(inputs, dim))
 
+    def test_exponential(self):
```

Comment:
maybe comment that the point of this test is to check that ops that take in `Float` will specialize instead of erroring on dynamic shapes? (and `exponential_` happens to take in a `Float` today, but that may change).